client/sites/components/sites-dataviews/dataviews-fields/site-field.tsx

```diff
@@ -60,7 +63,49 @@ const ListTileTitle = styled.div`
 	align-items: center;
 `;
 
-const SiteField = ( { site, openSitePreviewPane }: Props ) => {
+/**
+ * Renders an anchor element that can trigger analytics calls
+ * or client-side router navigation via the `onNavigate` prop.
+ */
+const Link = ( {
+	disabled,
+	href,
+	onNavigate,
+	...props
+}: {
+	/** Accessibly disable the link. Use sparingly. */
+	disabled?: boolean;
```

Comment:
I'm tempted to leave this `disabled` feature out of the component because it really shouldn't be used often, but then again the accessibility details are tricky to get right if we leave this up to ad hoc implementations 🙁 I guess we'll see if it gets overused.

Reply:
How about a separate util to "disable" properly, that can be used as an extra if needed?

Reply:
Could be a possibility. We'll see how many disabled faux links we come across, and if they're rare to begin with, maybe we can just leave it out.

Reply:
Sounds good. Also, I like the "accessible while disabled" behaviour as the default and only alternative